src/data/vapor-pressure-constants.ts

```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
+    CAS: "110-74-7",
+    C1: 104.08,
+    C2: -7535.9,
+    C3: -12.348,
+    C4: 0.000009602,
+    C5: 2,
+    minimumTemperature: 180.25,
+    maximumTemperature: 538,
+    pressureAtMinimumTemperature: 2.11e-1,
+    pressureAtMaximumTemperature: 4.031e6
+  },
+
+  "2PropylMercaptan": {
+    name: "2-Propyl mercaptan",
+    formula: "C3H8S",
+    CAS: "75-33-2",
+    C1: 60.43,
+    C2: -5276.9,
+    C3: -5.6572,
+    C4: 2.6e-17,
+    C5: 6,
+    minimumTemperature: 142.61,
+    maximumTemperature: 517,
+    pressureAtMinimumTemperature: 9.73e-3,
+    pressureAtMaximumTemperature: 4.752e6
+  },
+
+  PropylMercaptan: {
+    name: "Propyl mercaptan",
+    formula: "C3H8S",
+    CAS: "107-03-9",
+    C1: 62.165,
+    C2: -5624,
+    C3: -5.8595,
+    C4: 2.06e-17,
+    C5: 6,
+    minimumTemperature: 159.95,
+    maximumTemperature: 536.6,
+    pressureAtMinimumTemperature: 6.51e-2,
+    pressureAtMaximumTemperature: 4.627e6
+  },
+
+  "1,2PropyleneGlycol": {
+    name: "1,2-Propylene glycol",
+    formula: "C3H8O2",
+    CAS: "57-55-6",
+    C1: 212.8,
+    C2: -15420,
+    C3: -28.109,
+    C4: 0.000021564,
+    C5: 2,
+    minimumTemperature: 213.15,
+    maximumTemperature: 626,
+    pressureAtMinimumTemperature: 9.29e-5,
+    pressureAtMaximumTemperature: 6.041e6
+  },
+
+  Quinone: {
+    name: "Quinone",
+    formula: "C6H4O2",
+    CAS: "106-51-4",
+    C1: 48.651,
+    C2: -7289.5,
+    C3: -3.4453,
+    C4: 1.01e-18,
+    C5: 6,
+    minimumTemperature: 388.85,
+    maximumTemperature: 683,
+    pressureAtMinimumTemperature: 1.17e4,
+    pressureAtMaximumTemperature: 5.925e6
+  },
+
+  SiliconTetrafluoride: {
+    name: "Silicon tetrafluoride",
+    formula: "F4Si",
+    CAS: "7783-61-1",
+    C1: 272.85,
+    C2: -9548.9,
+    C3: -40.089,
+    C4: 6.37e-15,
+    C5: 6,
+    minimumTemperature: 186.35,
+    maximumTemperature: 259,
+    pressureAtMinimumTemperature: 2.21e5,
+    pressureAtMaximumTemperature: 3.748e6
+  },
+
+  Styrene: {
+    name: "Styrene",
+    formula: "C8H8",
+    CAS: "100-42-5",
+    C1: 105.93,
+    C2: -8685.9,
+    C3: -12.42,
+    C4: 7.5583e-6,
+    C5: 2,
+    minimumTemperature: 242.54,
+    maximumTemperature: 636,
+    pressureAtMinimumTemperature: 1.06e1,
+    pressureAtMaximumTemperature: 3.823e6
+  },
+
+  SuccinicAcid: {
+    name: "Succinic acid",
+    formula: "C4H6O4",
+    CAS: "110-15-6",
+    C1: 165.977,
+    C2: -19914.4,
+    C3: -18.9344,
+    C4: 1.91e-18,
+    C5: 6,
+    minimumTemperature: 460.85,
+    maximumTemperature: 838,
+    pressureAtMinimumTemperature: 7.78e2,
+    pressureAtMaximumTemperature: 5.001e6
+  },
+
+  SulfurDioxide: {
+    name: "Sulfur dioxide",
+    formula: "O2S",
+    CAS: "7446-09-5",
+    C1: 47.365,
+    C2: -4084.5,
+    C3: -3.6469,
+    C4: 1.8e-17,
+    C5: 6,
+    minimumTemperature: 197.67,
+    maximumTemperature: 430.75,
+    pressureAtMinimumTemperature: 1.67e3,
+    pressureAtMaximumTemperature: 7.86e6
+  },
+
+  SulfurHexafluoride: {
```

Comment:
@maneetgoyal @sumitnegi7  we need to change interface accordingly for SulfurHexafluoride , some of its data is missing